Dockerfile.template

```diff
@@ -22,99 +22,21 @@ RUN set -eux; \
 	apk add --no-cache \
 # add tzdata for https://github.com/docker-library/redis/issues/138
 		tzdata \
+# add setpriv for step down from root.
+		setpriv \
 	;
 {{ ) else ( -}}
 RUN set -eux; \
 	apt-get update; \
 	apt-get install -y --no-install-recommends \
 # add tzdata explicitly for https://github.com/docker-library/redis/issues/138 (see also https://bugs.debian.org/837060 and related)
 		tzdata \
+# add util-linux for setpriv for step down from root.
+		util-linux \
```

Comment:
This is already part of the base image (so we do not need to explicitly install it).